src/lib/mongoBackend/mongoRegistrationGet.cpp

```diff
@@ -59,11 +59,11 @@ static void setRegistrationId(ngsiv2::Registration* regP, const mongo::BSONObj&
 *
 * setDescription -
 */
-static void setDescription(ngsiv2::Registration* regP, const mongo::BSONObj& r)
+static void setDescription(ngsiv2::Registration* regP, const mongo::BSONObj& r, const char* dbLabel)
```

Comment:
Function is overparametriced, `dbLabel` is not needed at the field used for descriptions is not a parameter but fixed by the database model.

Reply:
Ah, of course. Forgot this one ...

Reply:
Fixed in 5d851ef